vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,136 @@
+
+Git-based bundles do not need to be to upgraded. Both types of bundles are compatible on Entando 7.1+, but docker-based bundles provide several innovations and an improved developer experience.
+
+The improvements include:
+* Simplified bundle directory structure
+* A decoupled build and deployment process that is easier to understand, modify, and troubleshoot
+* CI/CD integration and support 
+* One Git source repository required with Docker image repositories created automatically as needed
+* Optimized file storage and transfer methods using a layered file system, beneficial for large projects
+* Standardized bundle address format
+
+## Quick Reference
+* [Bundle Specifications](../curate/bundle-details.md)
+
+* [Bundle Management](../getting-started/ent-bundle.md)
+
+* [Bundle Version Comparison Table](./bundle-comparison.md)
+
+## Upgrade Process
+
+### Initialize Bundle 
+
+1. Initialize a new bundle to set up the standard Entando Bundle structure using the ent CLI:
+
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+```
+### Copy Assets
+1. Use the ent CLI bundler command to export assets from an existing Entando Bundle. See the step-by-step [Export and Publish Tutorial](../../tutorials/create/pb/export-bundle-from-application.md). 
+
+2. `YOUR-BUNDLE-NAME/platform` directory is designated for resources such as fragments, contents, pages, and templates. Copy the output of the ent bundler to this folder. You can also accomplish this manually but there are advantages to the bundler such as generating top level descriptors for platform resources.
+Note that widgets that are not MFEs are considered platform entities on Entando and should be placed in the `platform/widgets` directory.
+
+### Add Micro Frontends & Microservices
+ 
+1. Add your micro frontend to the new bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+The docker-based bundle has a centralized bundle descriptor with bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management specifications. 
+
+2. Copy the MFE files into `YOUR-BUNDLE-NAME/microfrontends` directory. For instance, for an MFE named YOUR-MFE1: 
+
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MFE1/{.,}* microfrontends/YOUR-MFE1
+```
+Micro frontends and microservices are partitioned into their own directories in the docker-based bundles to streamline the CI/CD process.
+
+3. Add your microservice:
+``` sh
+ent bundle ms add YOUR-MS1
+```
+4. Copy the microservice YOUR-MS1 files to the new location:
+``` sh
+mv YOUR-ORIGIN-DIRECTORY/YOUR-MS1/{.,}* microservicess/YOUR-MS1
+```
+5. Repeat steps 1 through 4 for all micro frontends and microservices.
+
+### Add API Claims
+1. Now add an API claim to connect the `YOUR-MFE1` to a microsersive as needed. The connection information is added to the descriptor `entando.json`.
```

Comment:
"now" is not necessary, and no backticks for the placeholder (per our convention)

"to connect YOUR-MFE1" (no "the")